### PR TITLE
Fleshed out README.md. Added CONTRIBUTING.md.

### PR DESCRIPTION
**Pull request by amiller-gh**
See https://github.com/nodejs/community-committee/issues/212 for additional context.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,25 @@
 # Badges Initiative
+
+This is the organizational repository for the Node.js Badges Strategic Initiative, as chartered under the [Community Committee].
+
+This initiative is tasked with facilitating the design, development and production of Node.js Open Badges and associated content.
```
Comment (MylesBorins) [inline README.md]: can you include a definition of what `Open Badges` is? I see some context with responsibilities below, but as someone unfamilliar with this initiative it is not obvious just yet

**Review by MylesBorins — COMMENTED** [review]
I would like to see a license included in this PR as well.

Without a license many individuals from organizations with open source policies will be unable to contribute

File: README.md [hunk 2]
```diff
@@ -1 +1,25 @@
...
+ - Badge award processes;
+ - Development and Maintenance of badge issuance automation tooling;
+ - Production and distribution of stickers, pins, and related products created from content that emerges from this initiative.
+
+## Governance
+This initiative adopts the general Node.js [Code of Conduct], as well as its own
+[Contributing Guidelines].
+
+## Team
+- [@amiller-gh](https://github.com/amiller-gh) - **Adam Miller**, Community Committee Champion
```
Comment (amiller-gh) [inline README.md:20]: Its lonely here! Consider this PR an open call for team members who want to actively work on this project 👍 

Tagging all participants from the [inaugural issue](https://github.com/nodejs/community-committee/issues/212) here: @Bamieh, @bnb, @oe, @Tiriel, @rachelnicole, @gr2m, @mhdawson

**Comment by amiller-gh** [comment]
@MylesBorins, 👍 on the license. This seems to be an issue with may of the CommComm strategic initiative repos: 
https://github.com/nodejs/i18n
https://github.com/nodejs/website-redesign
https://github.com/nodejs/evangelism/
etc...

We may want to do a review of licenses in the Node.js Github org and open PRs for projects that don't have one.

File: README.md (hunk 1 above)
Comment (amiller-gh) [reply README.md]: Will do! The inaugural issue that inspired this new repo is here: https://github.com/nodejs/community-committee/issues/212

I'm planning to capture a lot of the info that lives in that issue in this repo over the next day or so – this PR is just to kick off the new initiative. Will update so it is more obvious to fresh eyes in the mean time though 🙂

File: README.md (hunk 2 above)
Comment (Tiriel) [reply README.md:20]: Count me in!

File: README.md (hunk 1 above)
Comment (MylesBorins) [reply README.md]: Rad thx, no need to block. Just wanted to share how this read without context

**Review by mhdawson — APPROVED** [review]
LGTM

File: README.md (hunk 2 above)
Comment (Tiriel) [reply README.md:20]: Sorry, I left this too long aside.
@amiller-gh Do you want me to submit a second PR to add myself when this one is merged, or do you prefer to add me now? If the latter, the name's Benjamin Zaslavsky!

File: README.md (hunk 1 above)
Comment (mcollina) [reply README.md]: I agree with @MylesBorins. Can you please add a description _why_ are we doing a open badges? Why are these important for our community?

File: README.md (hunk 2 above)
Comment (amiller-gh) [reply README.md:20]: Nope! I got ya. Sorry for the delay, busy week 👍

**Review by bnb — APPROVED** [review]
LGTM, pending any other requests 🤗

**Review by mhdawson — APPROVED** [review]
still LGTM

**Pull request merged**